.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "docs/jsdoc-template"]
 	path = docs/jsdoc-template
 	url = https://github.com/realm/realm-jsdoc.git
-[submodule "vendor/realm-core"]
-	path = vendor/realm-core
+[submodule "packages/realm/bindgen/vendor/realm-core"]
```

Comment:
`git mv` didn't change this line and I wasn't 100% sure if I should, but I decided to do it manually. Let me know if I shouldn't have.

Reply:
It worked wonderfully when I pulled and did a `git submodule update --init --recursive` locally.